Gemfile

```diff
@@ -73,9 +73,11 @@ group :development, :test do
   gem 'bummr'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'capybara'
+  gem 'debase', '~> 0.2.4.1'
   gem 'fcrepo_wrapper'
   gem 'rspec-rails'
   gem 'rspec_junit_formatter'
+  gem 'ruby-debug-ide'
```

Comment:
This gem, and possibly the others, should not be run in the production environment, please move the testing gems to the "development, testing" gem group lower in this file.

Reply:
Tao - my apologies, I was misreading the file because of the way git was displaying the diff.  I'm sorry for the delay.